<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fops</class>
 <widget class="QWidget" name="fops">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fops{border-image: url(:/images/menu1.jpg);}

QPushButton {
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);
	background-color: rgba(0, 33, 68,220);
	border:none; 
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
  </property>
  <widget class="QPushButton" name="back_menu_bt">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>191</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 33, 68,200);</string>
   </property>
   <widget class="QPushButton" name="video_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>录像</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/video.png</normaloff>:/images/video.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>29</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="photo_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>图片</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/photo.png</normaloff>:/images/photo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_edt">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>550</y>
     <width>113</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rename_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>重命名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rename_edt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>113</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>550</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string> 搜索</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>800</width>
     <height>490</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 33, 68,200);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="vidWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 33, 68,200);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="picWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 33, 68,200);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
